test/unit/rules/no-forbidden-elements-test.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+const ERROR_MESSAGE_FORBIDDEN_ELEMENTS = require('../../../lib/rules/no-forbidden-elements')
+  .ERROR_MESSAGE_FORBIDDEN_ELEMENTS;
+
+generateRuleTests({
+  name: 'no-forbidden-elements',
+
+  config: true,
+
+  good: ['<header></header>', '<div></div>', '<footer></footer>', '<p></p>'],
```

Comment:
Can you add some `good` examples that specify custom config to allow `<script>`?

Something like:

```js
{
  config: ['meta'],
  template: '<script></script>',
}
```

Reply:
Done